.bazelrc

```diff
@@ -319,10 +319,11 @@ build:compdb --build_tag_filters=-nocompdb
 
 # Windows build quirks
 build:windows --action_env=TMPDIR
+build:windows --action_env=INCLUDE
+build:windows --action_env=LIB
```

Comment:
It will be interesting to see if we want this, it's a change I can see locally, and tweak the docker container for the correct paths. Bazel doesn't need these, because it generates them itself.

Reply:
Seems to work fine without them.